tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "django-pipeline-tests",
```

Comment:
This file should include `"private": true` to prevent any accidental publication on npm.